rules/terraformrules/terraform_empty_list_equality_test.go

```diff
@@ -0,0 +1,107 @@
+package terraformrules
+
+import (
+	"testing"
+
+	hcl "github.com/hashicorp/hcl/v2"
+	"github.com/terraform-linters/tflint/tflint"
+)
+
+func Test_TerraformEmptyListCheckRule(t *testing.T) {
+	cases := []struct {
+		Name     string
+		Content  string
+		Expected tflint.Issues
+	}{
+		{
+			Name: "comparing with [] is not recommended",
```

Comment:
A simpler test would just be `count = [] == [] ? 0 : 1`. The rule only works on files with variables in them and because that is also true in the test cases that isn't caught by the tests. Keeping the test minimal helps prevent that situation.

Reply:
done. Replaced the two tests to just one with `count = [] == [] ? 0 : 1` that returns two results